packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -977,19 +954,15 @@ const handleEncryptionVersionMigration = () => async (dispatch: Dispatch, getSta
                 return; // should never happen
             }
 
-            // todo: argghhh
-            // eslint-disable-next-line no-nested-ternary
-            const nextData = result?.data
-                ? result.data
-                : entity.type === 'account'
-                ? ({
-                      accountLabel: '',
-                      outputLabels: {},
-                      addressLabels: {},
-                  } as AccountLabels)
-                : ({ walletLabel: '' } as WalletLabels);
-
-            console.debug('metadata migration: hydrating app state with: ', nextData);
+            const defaultEntityData =
+                entity.type === 'account'
+                    ? ({
+                          accountLabel: '',
+                          outputLabels: {},
+                          addressLabels: {},
+                      } as AccountLabels)
+                    : ({ walletLabel: '' } as WalletLabels);
```

Comment:
just a nitpick to think about. I think we are already defining this on a couple of places, maybe it could go to some single point of definition

Reply:
[3d81d0d](https://github.com/trezor/trezor-suite/pull/8825/commits/3d81d0d1500a2bd7b9ba98d4315cf6ed19f5cd5a)